tests/cephfs/clients/mds_scrub_after_failover.py

```diff
@@ -88,7 +88,7 @@ def run(ceph_cluster, **kw):
         fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
         fuse_mounting_dir_2 = f"/mnt/cephfs_fuse{mounting_dir}_2/"
         mon_node_ips = fs_util.get_mon_node_ips()
-        default_fs = "cephfs" if not erasure else "cephfs-ec"
+        default_fs = "cephfs-mds-scrub" if not erasure else "cephfs-mds-scrub-ec"
```

Comment:
Isn't default_fs and fs_name both same variables? Was there a specific reason to use 2 different variables here? Not a comment for this PR. Just checking on the original code owner to understand.

Reply:
Hi Neha,
No, it was the same. The code had few references, and I wanted to avoid disrupting them. So, I assigned a new filesystem name to default_fs as well.